torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -67,21 +88,21 @@ def add_kernel(self, kernel) -> int:
             return idx
 
     # Returns the triton kernel at the given index
-    def get_kernel(self, idx: int):
+    def get_kernel(self, idx: int) -> "TritonKernelType":
         # No need to lock here as fetching from dict is atomic
         assert idx in self.id_to_kernel
         return self.id_to_kernel[idx]
 
     # Not every constant arg can be added to the graph. Use this side table
     # for constant args.
-    def add_constant_args(self, args) -> int:
+    def add_constant_args(self, args: Any) -> int:
```

Comment:
Not your change but does `args` represent a single arg or multiple args? The name being plural confuses me... If it's actually multiple then it should probably take (and get below should return) a Sequence...

Reply:
I think this is actually a single arg, so I'll leave this as `Any` (cc @oulgen).

Reply:
Its a dict: https://fburl.com/code/7g7aurgi

so multiple items?

Reply:
actually, this is not a single arg - updating the typing